lib/iris/cube.py

```diff
@@ -62,7 +62,8 @@ class CubeMetadata(collections.namedtuple('CubeMetadata',
                                            'var_name',
                                            'units',
                                            'attributes',
-                                           'cell_methods'])):
+                                           'cell_methods',
+                                           'cell_measures_and_dims'])):
```

Comment:
@marqh This is a **really, really** fine point made by @pp-mo that I'd never considered ... and I agree with him in that cube metadata should be independent of cube dimensionality.

Perhaps there's no dodging the cube concatenate and cube merge bullets after all :cry: 

Are there any other opinions on whether dimensionality should be mixed into cube metadata, nudge @rhattersley @pelson et al.